.dockerignore

```diff
@@ -12,7 +12,7 @@ build
 **/.DS_Store
 
 # Docker
-docker-compose.*
+docker compose.*
```

Comment:
You don't need to update this file that reference file names not commands to execute.

Reply:
Thank you for your suggestion and explanation :) 
I'll update my commit that as soon as possible